crypto/bls_thresholdsign.go

```diff
@@ -37,6 +33,8 @@ import (
 
 // blsThresholdSignatureParticipant implements ThresholdSignatureParticipant
 // based on the BLS signature scheme
+var _ ThresholdSignatureParticipant = (*blsThresholdSignatureParticipant)(nil)
+
```

Comment:
```suggestion
var _ ThresholdSignatureParticipant = (*blsThresholdSignatureParticipant)(nil)

// blsThresholdSignatureParticipant implements ThresholdSignatureParticipant
// based on the BLS signature scheme
```

godoc above documented type